src/python/pants/backend/python/target_types.py

```diff
@@ -152,6 +154,8 @@ class ResolvePexEntryPointRequest:
     2. The `entry_point` using a shorthand `:my_func`, and the `sources` field being set. We
         combine these into `path.to.module:my_func`.
     3. The `entry_point` being left off, but `sources` defined. We will use `path.to.module`.
+
+    Users can set `entry_point='<none>'` to leave off the entry point.
```

Comment:
Would be good to explain what the result of that would be, if possible.

Reply:
Will do in a followup.